uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -1,11 +1,11 @@
 package com.virtuslab.gitmachete.uitest
 
+import java.nio.file.Files
+import java.nio.file.attribute.FileTime
+
 import com.virtuslab.gitmachete.testcommon.BaseGitRepositoryBackedIntegrationTestSuite
 import com.virtuslab.gitmachete.testcommon.BaseGitRepositoryBackedIntegrationTestSuite.SETUP_WITH_SINGLE_REMOTE
-import org.junit.After
-import org.junit.Assert
-import org.junit.Before
-import org.junit.Test
+import org.junit.{After, Assert, Before, Test}
```

Comment:
While you're here... pls add a mention to development-guide.md that installing Scala plugin might be useful for editing UI tests